lib/popcorn/package.ini

```diff
@@ -0,0 +1,11 @@
+[package]
+name=popcorn
+tags=web,lib
+maintainer=Alexandre Terrasa <alexandre@moz-code.org>
+license=Apache-2.0
+[upstream]
```

Comment:
I you want your own version numbering, there is a `package.version` key if I remember correctly.